tensorboard/webapp/widgets/line_chart_v2/lib/renderer/BUILD

```diff
@@ -25,7 +25,7 @@ tf_ts_library(
         "renderer_types.ts",
```

Comment:
Can we visibility restrict this target to lib:subpackages?  `public_types.ts` only re-exports specific symbols from `renderer_types.ts`, suggesting that some of its symbols may be non-public.

Reply:
Done.